### PR TITLE
Removed flyway instatiation in app, it will be driven by db command

### PR DESCRIPTION
**Pull request by maheshkelkar**
For deploydb.example.yml, use persistent h2 instead of in-memory

References #99

### CODE REVIEW

**Comment by rtyler** [comment]
Will you update the README to reflect the "proper" setup steps?

File: src/main/groovy/deploydb/models/Artifact.groovy [hunk 1]
```diff
@@ -82,4 +82,11 @@ class Artifact extends AbstractModel {
         return Objects.hash(this.id, this.group, this.name,
                             this.version, this.sourceUrl)
     }
+
+    @Override
```
Comment (rtyler) [inline src/main/groovy/deploydb/models/Artifact.groovy]: This change has nothing to do with this commit and pull request

File: src/main/groovy/deploydb/models/Deployment.groovy [hunk 2]
```diff
@@ -131,7 +131,7 @@ class Deployment extends AbstractModel {
     @Override
     String toString() {
         return "id: ${id}, environment: ${environmentIdent}, service: ${serviceIdent}, " +
-                "status: ${status}, " + "promotionResultSet: ${promotionResultSet}, " +
+                "status: ${status}, promotionResultSet: ${promotionResultSet}, " +
```
Comment (rtyler) [inline src/main/groovy/deploydb/models/Deployment.groovy]: This change also has nothing to do with this change

File: src/main/groovy/deploydb/DeployDBApp.groovy [hunk 3]
```diff
@@ -89,14 +87,6 @@ class DeployDBApp extends Application<DeployDBConfiguration> {
     @Override
     public void run(DeployDBConfiguration configuration,
                     Environment environment) {
-        /* Execute migrations */
-        Flyway flyway = new Flyway()
```
Comment (rtyler) [inline src/main/groovy/deploydb/DeployDBApp.groovy]: :+1:

File: deploydb.example.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 database:
   driverClass: org.h2.Driver
   user: nobody
-  url: "jdbc:h2:mem"
+  url: "jdbc:h2:~/deployDB.example.file"
```
Comment (rtyler) [inline deploydb.example.yml]: Can you make this default to `./deploydb.example.db` instead, so when people run the example code they're not having files dumped in their home directory unexpectedly?

Comment (maheshkelkar) [inline deploydb.example.yml]: ok

File: src/main/groovy/deploydb/models/Artifact.groovy (hunk 1 above)
Comment (maheshkelkar) [inline src/main/groovy/deploydb/models/Artifact.groovy]: But, right thing to do, hence rolled it in :)

File: src/main/groovy/deploydb/models/Deployment.groovy (hunk 2 above)
Comment (maheshkelkar) [inline src/main/groovy/deploydb/models/Deployment.groovy]: But, right thing to do, hence rolled it in :)

**Pull request merged**